src/prefect/cache_policies.py

```diff
@@ -110,39 +110,56 @@ def compute_key(
         flow_parameters: Dict[str, Any],
         **kwargs,
     ) -> Optional[str]:
-        return str(task_ctx.task_run.id)
+        return None
 
 
 @dataclass
-class _None(CachePolicy):
-    "ignore key policies altogether, always run - prevents persistence"
-
+class TaskSource(CachePolicy):
```

Comment:
A docstring for this class explaining the behavior of this cache policy would be helpful for users.

Reply:
Will do!